### DISCUSSION TITLE
What the best way to play with multiple/new windows or tabs?

### DISCUSSION THREAD

**Discussion by zedtux** [rubycdp/cuprite#236] [General]
I need to write a Cucumber test to validate that when changing the password from my Rails application, other open sessions (in new window or new tab) are well closed.

What is the best way, with Cuprite, to manage windows or tabs? Creating, switching and counting them?

Thanks !

**Comment by route** [comment]
Hi, could you please try this approach https://github.com/rubycdp/ferrum/blob/04463bf016bee6408e985769405eebf06b6bdc04/spec/dialog_spec.rb#L20 and let me know, though I’m not sure how it would work in your scenario.
